docs/content/manual/_incoming/2563-image-naming-inline-css.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Image naming with inline CSS
+---
+
+* Related issues: [#2563](https://github.com/harvester/harvester/issues/2563) [[BUG] harvesterhci.io.virtualmachineimage spec.displayName displays differently in single view of image
+
+## Category: 
+* Images
+
+## Verification Steps
+1. Go to images
+1. Click "Create"
+1. Upload an image or leverage an url - but name the image something like:
+<strong><em>something_interesting</em></strong>
```

Comment:
Maybe we would want to escape the html/css example.
I noticed rendering it locally with Hugo I see:
![Screenshot from 2022-08-22 13-41-33](https://user-images.githubusercontent.com/5370752/186015056-8224cce4-6ad9-4de8-b41c-766016629939.png)

^seems to not render the tags.
Maybe we could just wrap it in a code block-kinda example in Hugo
with like " ``` " so something like,
```
<strong><em>something_interesting</em></strong>
```
So when the site renders, it won't parse out the tags.

Reply:
Done